examples/web-wagmi/src/misc/LoginSpecificProfile.tsx

```diff
@@ -0,0 +1,31 @@
+import { useActiveProfile } from '@lens-protocol/react';
+import { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+
+import { LoginButton } from '../components/auth/LoginButton';
+
+export function LoginSpecificProfile() {
+  const [handle, setHandle] = useState('');
+  const activeProfile = useActiveProfile();
+
+  return (
+    <>
+      {activeProfile.data && <Navigate to="/" />}
```

Comment:
instead of redirecting away from this page, better to display an infobox with sth like "you are already logged in as @handle, if you want to change your active profile select from the list below"